package.json

```diff
@@ -27,10 +27,10 @@
     "chalk": "^2.4.2",
     "esm": "^3.2.25",
     "listr": "^0.14.3",
-    "listr-verbose-renderer": "^0.5.0"
+    "listr-verbose-renderer": "^0.5.0",
+    "@sumup/foundry": "^2.2.0"
```

Comment:
Why did you move Foundry to dependencies?

Reply:
Because lint was complaining

Reply:
Where was it complaining? What was the error message?

Reply:
Oh, sorry.
Inside the foundry configuration file, was showing an error about foundry being on devDependencies

Reply:
Those config files should be ignored by Eslint in Foundry v2.2. Anyway 🤷‍♂️